File: app/src/main/java/shafi/example/flatmemberrecyclerview/MainActivity.java
package shafi.example.flatmemberrecyclerview;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    private RecyclerView memberRV;
    private MemberAdapter memberAdapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        memberRV = findViewById(R.id.memberRV);
        memberAdapter = new MemberAdapter(this, Member.memberArrayList());
        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(this);
        memberRV.setLayoutManager(linearLayoutManager);
        memberRV.setAdapter(memberAdapter);
    }
}

File: app/src/main/java/shafi/example/flatmemberrecyclerview/MainActivity.kt
package shafi.example.flatmemberrecyclerview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import shafi.example.flatmemberrecyclerview.R
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.LinearLayoutManager

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val memberRV = findViewById<RecyclerView>(R.id.memberRV)
        val memberAdapter = MemberAdapter(this, Member.memberArrayList())
        val linearLayoutManager = LinearLayoutManager(this)
        memberRV.layoutManager = linearLayoutManager
        memberRV.adapter = memberAdapter
    }
}